compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/fragments/EvaluatorFragmentInfo.kt

```diff
@@ -19,8 +19,14 @@ import org.jetbrains.kotlin.descriptors.FunctionDescriptor
  *  This data structure contains "synthesized" descriptors for that class,
```

Comment:
This comment seems now a bit out of place, since it specifically explains what `EvaluatorFramentInfo`'s `classDescriptor`, `methodDescriptor` and `parameters` are

Reply:
Absolutely right: it "drifted" away!

Fixed :)